src/gridstack.js

```diff
@@ -1156,6 +1164,9 @@
   GridStack.prototype._updateContainerHeight = function() {
     if (this.engine._batchMode) { return; }
     var row = this.engine.getRow();
+    if (row < this.opts.minRow) {
```

Comment:
yep, that is the right place to put it. as you can see you can also accomplish this with CSS `min-height` on the grid div, but setting minRow can be easier for some.